src/jit/valuenum.cpp

```diff
@@ -6883,7 +6887,6 @@ void Compiler::fgValueNumberTree(GenTree* tree)
         switch (oper)
         {
             case GT_LCL_VAR:
-            case GT_REG_VAR:
```

Comment:
Why do you delete this?

Reply:
I rebased and pushed to origin to fix this